docs/UserGuide.md

```diff
@@ -3,38 +3,45 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+RC4HDB is a **desktop app for managing RC4 housing related information, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, RC4HDB can be a convenient and intuitive way to facilitate your day-to-day workflow as a RC4 housing management staff.
+
+## Table of Contents
+
+* [Quick Start](#quick-start)
+* [Features](#features)
+    * [Viewing help : help](#viewing-help--help)
+    * [Adding a resident : add](#adding-a-resident--add)
+    * [Listing all residents : list](#listing-all-residents--list)
+    * [Editing an existing resident : edit](#editing-an-existing-resident--edit)
+    * [Locating residents by names : find](#locating-residents-by-name--find)
+    * [Filtering residents by field : filter](#filtering-residents-by-field--filter)
+    * [Deleting a resident : delete](#deleting-a-resident--delete)
+    * [Clearing all entries : clear](#clearing-all-entries--clear)
+    * [Exiting the program : exit](#exiting-the-program--exit)
+    * [Saving the data](#saving-the-data)
+    * [Editing the data file](#editing-the-data-file)
+    * [Importing from csv file : import](#importing-from-csv-file--import)
+    * [Exporting to csv file : export](#exporting-to-csv-file--export)
+    * [CSV file format](#csv-file-format)
```

Comment:
Missing the **Format for resident fields** section.

Reply:
Thanks for highlighting it to me. Updated! 👍